src/dashboard/main.py

```diff
@@ -62,4 +62,4 @@ def page_container() -> Component:
 
 
 if __name__ == "__main__":
-    app.run("127.0.0.1", PORT, debug=True)
+    app.run("0.0.0.0", PORT, debug=True)
```

Comment:
Don't commit stuff like this. Running server locally is supposed to serve for localhost only.